### PR TITLE
landing page dashboard

### PR DESCRIPTION
**Pull request by SamirTouira**

### CODE REVIEW

File: apps/dashboard/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  {
+    path: 'landing',
+    loadChildren: () => import('./landing/landing.module').then(m => m.LandingModule)
+  }
+]
+
+@NgModule({
+  declarations: [],
+  exports: [RouterModule],
```
Comment (fritzschoff) [inline apps/dashboard/src/app/app-routing.module.ts:23]: remove the export here

File: apps/dashboard/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  {
+    path: 'marketplace',
+    loadChildren: () => import('../../../marketplace/src/app/marketplace/marketplace.module').then(m => m.MarketplaceModule)
+  },
+  {
+    path: 'faq',
+    loadChildren: () => import('../../../marketplace/src/app/faq/faq.module').then(m => m.FAQModule)
+  },
+  {
+    path: 'signin',
```
Comment (fritzschoff) [inline apps/dashboard/src/app/app-routing.module.ts]: we dont need a signin route, same for faq and marketpalce and contact

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/app/app-routing.module.ts (hunk 2 above)
Comment (SamirTouira) [reply apps/dashboard/src/app/app-routing.module.ts]: same for 'account' as My-tokens?

Comment (fritzschoff) [reply apps/dashboard/src/app/app-routing.module.ts]: going to be a different component that @RemcoSimonides  is working on

Comment (phpgeekfr) [reply apps/dashboard/src/app/app-routing.module.ts]: >we dont need a signin route

So guards have to be updated also

File: apps/dashboard/src/app/app-routing.module.ts (hunk 1 above)
Comment (SamirTouira) [reply apps/dashboard/src/app/app-routing.module.ts:23]: why ? if i remove this, the router-outlet won't work

**Comment by fritzschoff** [comment]
closing from cherrypicked branch #59

**Pull request closed without merging**